app/src/components/blocks/_lloydGeorge/lloydGeorgeViewRecordStage/LloydGeorgeViewRecordStage.test.tsx

```diff
@@ -62,7 +62,9 @@ describe('LloydGeorgeViewRecordStage', () => {
         expect(screen.getByText(`Last updated: ${mockPdf.lastUpdated}`)).toBeInTheDocument();
 
         expect(
-            screen.queryByText('No documents are available for this patient.'),
+            screen.queryByText(
+                'This patient does not have a Lloyd George record stored in this service for this patient.',
```

Comment:
Shouldn't this be:
```suggestion
                'This patient does not have a Lloyd George record stored in this service.',
```
?